app/Role.php

```diff
@@ -7,12 +7,12 @@
 class Role extends Model
 {
     public $timestamps = false;
-    protected $with = ['users:id,full_name'];
+    protected $with = ['users'];
```

Comment:
why do you need the whole user object???

Reply:
it wasn't working. The user array was empty until i removed that. I tried figuring what is wrong